### DISCUSSION TITLE
Feature request - [do not] Split multi-frame images

### DISCUSSION THREAD

**Discussion by bkossows** [bettar/miele-lxiv#110] [Ideas]
Hi,
We've been using Osirix for a quite long in our lab to convert images using in-house plugin (https://github.com/mslw/horos-bids-output). I've just realized that a free Lite version will be no longer available after trial period and we are stuck..
We'd gladly switch to Miele but the software does not comply with the new EPI DICOMs from Siemens Prisma. By default the scans are now marked as multi-frames and they would appear as separate series (as many as number of volumes!) in the Database window. Please add the feature to not split multi-frames images as it has been available in Osirix.
![multi-frame](https://github.com/bettar/miele-lxiv/assets/28865905/4ee839a1-d2a8-40f3-841e-a89867f607f0)
We will be happy to support Your project!
B.

**Comment by bettar** [comment]
Can you please make available to me (maybe privately) some samples of those EPI DICOM files from Siemens Prisma, for me to use during development and testing ?
